src/mathematics/glyphinfo.rs

```diff
@@ -0,0 +1,142 @@
+use truetype::{Result, Tape, Value};
+
+use layout::Coverage;
+use super::ValueRecord;
+
+table! {
+    @position
+    #[doc = "A table of positioning information defined on a per-glyph bases."]
+    pub GlyphInfo { // MathGlyphInfo
+        italics_correction_offset      (u16), // MathItalicsCorrectionInfo
+        top_accent_offset              (u16), // MathTopAccentAttachment
+        extended_shape_coverage_offset (u16), // ExtendedShapeCoverage
+        kern_info_offset               (u16), // MathKernInfo
+
+        italics_corrections (Option<ItalicsCorrections>) |this, tape, position| {
```

Comment:
I originally had `kern_info` and `italics_corrections` as required, but later found out during testing that LatinModern-Math doesn't provide kerning information in the math table.  Without kerning being optional LatinModern-Math -- and so I suspect all of the GUST created fonts -- will panic.  After learning this, I felt less confident about whether or not fonts supported the other tables as well.  I'm not sure what to do about this situation to be honest.